_posts/2020-09-21-changelog-131.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #131"
+date: 2020-09-17 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6039 -->
+* Provide intention to create a method from an unresolved method call (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-131/create_method" w="650px" %}
```

Comment:
Looks like it's the same gif as in PR. Unfortunately, it's quite blurry. Let's create a new one